bot/bot.py

```diff
@@ -1,12 +1,38 @@
-from tokenize import Token
 import hikari
 import os
 from dotenv import load_dotenv
 
-load_dotenv()
-intents = hikari.Intents.GUILD_VOICE_STATES | hikari.Intents.GUILD_SCHEDULED_EVENTS
-intents ^= 
+import lightbulb
+
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import db 
+
+from config import *
```

Comment:
When including third-party modules you should include them in `requirements.txt` example: `hikari-lightbulb==2.2.4`

also I don't recommend using cfg file to store tokens and and secret keys, we use environment variables for that